addon/components/paper-menu-content-inner.js

```diff
@@ -54,12 +54,12 @@ export default Component.extend(ParentMixin, {
   },
 
   focusMenuItem(e, direction) {
-    let currentItem = this.$(e.target).closest('md-menu-item');
+    let currentItem = e.target.closest('md-menu-item');
```

Comment:
unfortunately, we can't use `closest` as that is not supported in IE11.
We're trying to keep IE11 support because ember also does.

With that being said, I wouldn't mind if we added a polyfill to our dependencies: https://github.com/jonathantneal/closest

Then we `app.import()` the file in our `index.js` and we can start using closest anywhere.